doc/source/ray-observability/ray-logging.rst

```diff
@@ -4,6 +4,38 @@ Logging
 =======
 This document will explain Ray's logging system and its best practices.
 
+Internal Ray Logging Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When ``import ray`` is executed, Ray's logging system is initialized, generating a sensible configuration given in ``python/ray/log.py``. The default logging level is ``logging.INFO``.
+
+Ray uses `rich <https://github.com/Textualize/rich>`__ to generate log messages that are well formatted and easily identified. If ``rich`` is not installed, a simple fallback formatter is used. Rich log messages display contextual information including the Ray component from which the message was generated and the filename and line number.
+
+Changing Ray's Internal Logging Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To modify Ray's logging configuration, change the configuration in ``python/ray/log.py`` according to `the python logging documentation <https://docs.python.org/3/howto/logging.html>`__. Ray uses a `dictConfig <https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig>`__ to specify logging configuration.
+
+Logging configuration should be kept inside ``python/ray/log.py`` to avoid spreading conflicting logging configurations across the database. There is no need to call ``logging.basicConfig`` because logging configuration is done at import time.
+
+.. note::
+
+   The exception is if you need logging configuration in a separate process. Because the Python logger module creates a singleton logger per process, if a worker needs to log a message it should configure logging on a per task or actor basis. For more information, see :ref:`Customizing worker loggers <customizing-worker-loggers> section.`
+
+Additionally, do not log Ray log messages to the root logger. `Always to a logger with the current file name <https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial>`__:
```

Comment:
This doesn't seem to be necessary for users to know. Can we remove  this section?